### DISCUSSION TITLE
Option to "not-purge" support

### DISCUSSION THREAD

**Discussion by matthewpull** [tailwindlabs/tailwindcss#7005] [Ideas]
Hi :wave: 

Absolutely love the new Tailwind v3 update - tonnes of new features and some significant updates, really good work!

One minor quarm I have is that the "not-prose" feature means that all of my prose selectors - and I have a lot of them - are a fair bit longer.

My project doesn't contain any `.not-prose` element, so for me all this does is add file size.

What I would love is an option is disable not-prose support, so we can have the old class selectors.

This is only minor, but I think would make quite a difference to those not using `.not-prose`.

Thanks!

**Comment by matthewpull** [comment]
Fyi, if you add the plugin to your `tailwind.config.js` like this:

```js
plugins: [
  require('@tailwindcss/typography')({
    target: 'legacy',
  }),
],
```

then that seems to achieve the effect I'm looking for.

However, based on the name "legacy" and the fact that I can't find this documented anywhere, I'm not convinced that this won't be deprecated / removed, or will cause other issues - if you use this make sure to test thoroughly!

**Reply by daviareias** [reply]
I was building an email with maizzle, and had to search deep into the files to find that you had that option to make it work.

Not sure why it's not in the documentation :(
